data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -143,64 +171,94 @@ def _climate_factor(self) -> bool:
+                field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        return (
+        expected_agriculture_loss_threshold = (
             self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
                 + field_names.PERCENTILE_FIELD_SUFFIX
             ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & climate_criteria
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        expected_building_loss_threshold = (
+            self.df[
+                field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        self.df[field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD] = (
+            expected_population_loss_threshold
+            & self.df[field_names.FPL_200_SERIES]
+        )
+
+        self.df[field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD] = (
+            expected_agriculture_loss_threshold
+            & self.df[field_names.FPL_200_SERIES]
+        )
+
+        self.df[field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD] = (
+            expected_building_loss_threshold
+            & self.df[field_names.FPL_200_SERIES]
+        )
+
+        self._increment_total_eligibility_exceeded(climate_eligibility_columns)
+
+        return self.df[climate_eligibility_columns].any(axis="columns")
 
     def _energy_factor(self) -> bool:
         # In Xth percentile or above for DOE’s energy cost burden score (Source: LEAD Score)
         # AND
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
-        energy_criteria = (
+
+        energy_eligibility_columns = [
+            field_names.PM25_EXPOSURE_LOW_INCOME_FIELD,
+            field_names.ENERGY_BURDEN_LOW_INCOME_FIELD,
+        ]
+
+        energy_burden_threshold = (
             self.df[
                 field_names.ENERGY_BURDEN_FIELD
                 + field_names.PERCENTILE_FIELD_SUFFIX
             ]
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-        ) | (
+        )
+
+        pm25_threshold = (
             self.df[
                 field_names.PM25_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
             ]
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & energy_criteria
+        self.df[field_names.PM25_EXPOSURE_LOW_INCOME_FIELD] = (
```

Comment:
You got these two backwards. `field_names.PM25_EXPOSURE_LOW_INCOME_FIELD` is referring to energy burden, and `field_names.ENERGY_BURDEN_LOW_INCOME_FIELD` is referring to pm2.5

Reply:
!!!!!!

Reply:
Thanks